File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/FremskrevetPensjonUnderUtbetaling.java
package no.nav.domain.pensjon.regler.beregning2011;

import no.nav.domain.pensjon.regler.kode.FormelKodeCti;

import java.io.Serializable;

public class FremskrevetPensjonUnderUtbetaling implements Serializable {
    
    private double pensjonUnderUtbetalingPerAr;
    private int gap;
    private double reguleringsfaktor;
    private FormelKodeCti formelKode;

    public FremskrevetPensjonUnderUtbetaling() {
        super();
        setFormelKode(new FormelKodeCti("BPUx"));
    }

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/FremskrevetPensjonUnderUtbetaling.kt
package no.nav.pensjon.regler.domain.beregning2011

import no.nav.pensjon.regler.domain.enum.FormelKodeEnum
import java.io.Serializable

class FremskrevetPensjonUnderUtbetaling : Serializable, Regulering {
    var pensjonUnderUtbetalingPerAr = 0.0
    override var gap = 0
    override var reguleringsfaktor = 0.0
    var formelKodeEnum: FormelKodeEnum = FormelKodeEnum.BPUx
}